client/src/utils/BGraphQueryUtil.ts

```diff
@@ -1,9 +1,10 @@
 import { Filter } from '@/types/typesLex';
 import { BIO_EDGE_TYPE_OPTIONS } from '@/utils/ModelTypeUtil';
+import { QUERY_FIELDS_MAP } from '@/utils/QueryFieldsUtil';
 
-// PATH QUERIES
+// BIO PATH QUERIES
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export function bioPathPre (bgraphPathQuery: any, clause: Filter): any {
+export function bioPathPre (bgraphPathQuery, clause: Filter): any {
```

Comment:
What the `Pre` stands for?

Reply:
Prefix and Post is Postfix. The code organization here is to enable query building; these helper functions are used to build up a bgraph query. 

If your query is a `bio path query` you'd prefix it using this helper and then postfix it using the `bioPathPost` helper.